index.html

```diff
@@ -235,5 +244,35 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
       </ul>
       <h3>diegoyon@outlook.com</h3>
     </footer>
+    <div class="desktop-footer">
+      <hr>
+      <ul class="social">
+        <li>
+          <a href="#">
+            <img src="./images/twitter.svg" alt="twitter" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/linkedin.svg" alt="linkedin" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/medium.svg" alt="medium" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/github.svg" alt="github" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/angellist.svg" alt="angellist" />
+          </a>
+        </li>
+      </ul>
+    </div>
```

Comment:
- [ ] it would be better if you remove the unused white space at the bottom of the page in order to make the portfolio clean and professional 
![image](https://user-images.githubusercontent.com/61790587/171491599-d0529a61-b92b-49b2-8018-b70621f042ae.png)